nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -141,32 +143,8 @@ define([
             busEventManager = BusEventManager.make({
                 bus: runtime.bus(),
             }),
-            typesToFiles = setupFileData(options.importData);
```

Comment:
I did some rearranging here to keep the class variable declarations mostly separate from the function declarations.